src/main/scala/com/databricks/labs/overwatch/pipeline/Pipeline.scala

```diff
@@ -142,7 +142,7 @@ class Pipeline(_workspace: Workspace, _database: Database,
       spark.catalog.tableExists(config.databaseName, "pipeline_report")) {
       val w = Window.partitionBy('organization_id, 'moduleID).orderBy('Pipeline_SnapTS.desc)
       spark.table(s"${config.databaseName}.pipeline_report")
-        .filter('Status === "SUCCESS" || 'Status.startsWith("EMPTY"))
+        .filter('status === "SUCCESS" || 'status.startsWith("EMPTY"))
```

Comment:
lower case "s" required. If failure occurs in spark_events_bronze where the spark SQL column case sensitivity is enabled, this could error.